systems/controllers/osc/operational_space_control.cc

```diff
@@ -248,67 +271,68 @@ void OperationalSpaceControl::Build() {
   // 2. Holonomic constraint
   holonomic_constraint_ =
       prog_->AddLinearEqualityConstraint(MatrixXd::Zero(n_h_, n_v_),
-               VectorXd::Zero(n_h_), dv_)
+                                         VectorXd::Zero(n_h_), dv_)
            .evaluator()
            .get();
   // 3. Contact constraint
-  if (body_indices_.size() > 0) {
+  if (all_contacts_.size() > 0) {
     if (w_soft_constraint_ <= 0) {
       contact_constraints_ =
-          prog_->AddLinearEqualityConstraint(MatrixXd::Zero(n_c_, n_v_),
-                   VectorXd::Zero(n_c_), dv_)
+          prog_->AddLinearEqualityConstraint(MatrixXd::Zero(n_c_active_, n_v_),
+                                             VectorXd::Zero(n_c_active_), dv_)
                .evaluator()
                .get();
     } else {
       // Relaxed version:
-      contact_constraints_ = prog_->AddLinearEqualityConstraint(
-                                      MatrixXd::Zero(n_c_, n_v_ + n_c_),
-                                      VectorXd::Zero(n_c_), {dv_, epsilon_})
-                                  .evaluator()
-                                  .get();
+      contact_constraints_ =
+          prog_->AddLinearEqualityConstraint(
+                   MatrixXd::Zero(n_c_active_, n_v_ + n_c_active_),
+                   VectorXd::Zero(n_c_active_), {dv_, epsilon_})
+               .evaluator()
+               .get();
     }
   }
-  // 4. Friction constraint (approximated firction cone)
-  if (!body_indices_.empty()) {
+  // 4. Friction constraint (approximated friction cone)
+  if (!all_contacts_.empty()) {
     VectorXd mu_minus1(2);
```

Comment:
we should rename `mu_minus1` and  `mu_plus1` to `mu_neg1` and `mu_1` because `mu_minus1` sounds like it is `(mu - 1)` instead of `mu, -1`

Reply:
done